addons/mail/wizard/mail_compose_message.py

```diff
@@ -516,17 +516,23 @@ def _process_state(self, mail_values_dict):
 
         return mail_values_dict
 
-    def _get_blacklist_record_ids(self, mail_values_dict):
+    def _get_blacklist_record_ids(self, mail_values_dict, recipients_info):
```

Comment:
When modifying an API in stable (even a private one), better make the argument optional (like recipients_info=None if it is a dict). That way if people are calling it in some custom code, it won't crash. They will just not benefit from the "improved" behavior when they call super() without the argument, but they won't suddenly suffer from crashes :) .